packages/@ember/-internals/glimmer/lib/components/link-to.ts

```diff
@@ -614,32 +614,13 @@ if (EMBER_MODERNIZED_BUILT_IN_COMPONENTS) {
     });
   }
 
-  // @disabledWhen
   {
     let superIsSupportedArgument = prototype['isSupportedArgument'];
 
     Object.defineProperty(prototype, 'isSupportedArgument', {
       configurable: true,
       enumerable: false,
       value: function isSupportedArgument(this: LinkTo, name: string): boolean {
-        if (this.modernized) {
-          if (name === 'disabledWhen') {
-            deprecate(
-              'Passing the `@disabledWhen` argument to <LinkTo> is deprecated. ' +
-                'Use the `@disabled` argument instead.',
-              false,
-              {
-                id: 'ember.link-to.disabled-when',
-                for: 'ember-source',
-                since: {},
-                until: '4.0.0',
-              }
-            );
-
-            return true;
-          }
-        }
-
         return superIsSupportedArgument.call(this, name);
       },
     });
```

Comment:
I think *all* of this can go, as what remains after your patch is only the call of the super method?